com.vagueentertainment.Framed.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Framed
+GenericName=Digital Picture Frame
+Comment=Digital picture frame for NFTshowroom and personal images
+Categories=Graphics;
+Exec=framed
+Icon=com.vagueentertainment.Framed
+Terminal=false
+Type=Application
+Keywords=Python;Godot;
```

Comment:
Same, some Godot leftovers.

Reply:
That's what I get for copying and pasting.